### PR TITLE
Tests/m2 4049 shared comp tests

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

File: src/shared/components/Pin/Pin.test.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+
+import { Pin } from './Pin';
+
+const mockOnClick = jest.fn();
+
+describe('Pin component tests', () => {
+  test('clicking the close button hides the banner', async () => {
```
Comment (IhorNesterenko99) [inline src/shared/components/Pin/Pin.test.tsx]: is it correct description?

File: src/shared/components/Modal/Modal.test.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  thirdBtnText: 'thirdBtnText',
+  hasSecondBtn: true,
+  hasThirdBtn: true,
+};
+
+describe('Modal Component', () => {
+  test('should render the modal with all elements', () => {
+    render(<Modal {...commonProps} />);
+
+    expect(screen.getByText(commonProps.title)).toBeInTheDocument();
```
Comment (wildkiomi) [inline src/shared/components/Modal/Modal.test.tsx:30]: texts.forEach(text => expect(screen.getByText(text)).toBeInTheDocument());

you could do smth like that

**Review by alex-i-e — APPROVED** [review]

File: src/shared/components/Pin/Pin.test.tsx (hunk 1 above)
Comment (kdanilenka32) [reply src/shared/components/Pin/Pin.test.tsx]: fixed

**Review by alex-i-e — CHANGES_REQUESTED** [review]
Need to update test description

File: src/shared/components/Banner/Banner.test.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+
+import { Banner } from './Banner';
+import { BANNER_LINK } from './Banner.const';
+
+describe('Banner component tests', () => {
```
Comment (alex-i-e) [inline src/shared/components/Banner/Banner.test.tsx]: No need to add `component tests` here and below, just source file name is enought.

**Review by muhurov — APPROVED** [review]

File: src/shared/components/Banner/Banner.test.tsx (hunk 3 above)
Comment (kdanilenka32) [reply src/shared/components/Banner/Banner.test.tsx]: fixed

**Review by alex-i-e — APPROVED** [review]

**Pull request merged**